pkg/ddc/alluxio/worker_test.go

```diff
@@ -168,23 +184,30 @@ func TestSetupWorkers(t *testing.T) {
 				namespace:   tt.fields.namespace,
 				Log:         ctrl.Log.WithName(tt.fields.name),
 			}
+
+			e.Helper = ctrlhelper.BuildHelper(tt.fields.runtimeInfo, mockClient, e.Log)
 			err := e.SetupWorkers()
 			if err != nil {
 				t.Errorf("AlluxioEngine.SetupWorkers() error = %v", err)
 			}
-			for _, node := range tt.fields.nodeInputs {
-				newNode, err := kubeclient.GetNode(mockClient, node.Name)
-				if err != nil {
-					t.Errorf("fail to get the node with the error %v", err)
-				}
 
-				if len(newNode.Labels) != len(tt.wantedNodeLabels[node.Name]) {
-					t.Errorf("fail to decrease the labels, newNode labels is %v", newNode.Labels)
-				}
-				if len(newNode.Labels) != 0 && !reflect.DeepEqual(newNode.Labels, tt.wantedNodeLabels[node.Name]) {
-					t.Errorf("fail to decrease the labels, newNode labels is %v", newNode.Labels)
-				}
+			if tt.fields.replicas != *tt.fields.worker.Spec.Replicas {
+				t.Errorf("Failed to scale %v for %v", tt.name, tt.fields)
 			}
+
+			//for _, node := range tt.fields.nodeInputs {
```

Comment:
If no, delete it

Reply:
thanks, delete it  will make the code cleaner~